deploy/tower-operator.yaml

```diff
@@ -113,14 +113,14 @@ spec:
             - /usr/local/bin/ao-logs
             - /tmp/ansible-operator/runner
             - stdout
-          image: "geerlingguy/tower-operator:0.3.0"
+          image: "ansible/tower-operator:latest"
```

Comment:
@matburt - Can you push up a version `0.4.0` tag, that way if someone installs the operator today and the underlying image at `:latest` is different in a month or two, it won't break from under them.

Reply:
Nevermind, there is now a `0.4.0` tag :)